requirements-dev.txt

```diff
@@ -1,2 +1,6 @@
 pytest==8.3.3
+mypy==1.11.2
+ruff==0.6.8
+isort==5.13.2
+
```

Comment:
They are not needed as explicit dependencies. The ruff and isort versions are already defined by `.pre-commit-config.yaml`.

To run them follow the instructions in [CONTRIBUTING.md](https://github.com/Akkudoktor-EOS/EOS/blob/main/CONTRIBUTING.md#code-style)

Reply:
done